### DISCUSSION TITLE
Use snapshots from camera directly

### DISCUSSION THREAD

**Discussion by ministryofsillywalks** [mmcc-xx/WhosAtMyFeeder#3] [General]
Do I have to go through frigate? I have a solar powered camera that is pir activated and so far didn't really have a use for it.
My idea was to set this up to watch a feeder. However, as the camera is battery powered it doesn't support direct streaming to frigate as its only accessible via its proprietary app.
However it can Mail image snapshots of detected movement. So my idea was to just feed these images into WhosAtMyFeeder, but these of course would be larger and contain more of the image than just the picture of the bird thats output via frigate.
Would this still work?

**Comment by mmcc-xx** [comment]
At the moment, it would not. But, I think it is a function worth pursuing.

For video streams, I think working on top of frigate makes sense as it does the first pass "is there a bird?" analysis. It also tracks events, which I'm planning to incorporate so that a given detection can link back to stored video of that detection.

However I think there is room for supporting some mechanism for analyzing stills. 

Might be a while before I get to it :)
